openadapt/crud.py

```diff
@@ -146,6 +146,10 @@ def get_recording(timestamp):
     return db.query(Recording).filter(Recording.timestamp == timestamp).first()
 
 
+def get_recording_by_id(id: int):
+    return db.query(Recording).filter(Recording.id == id).first()
+    
+
 def _get(table, recording_timestamp):
```

Comment:
WDYT about adding `get_recording_by_task_description` ?

Reason:

- Most of the time, when I work with fine-tuning, I need to point to a specific record and I never know the `id`, while description is more user-friendly as it is a string given by user.

Reply:
I think this is related to #430 , which would require fine tuning another model on recordings with their labels/completions as the respective task being carried out.